patches/stylelint+13.6.1.patch

```diff
@@ -0,0 +1,28 @@
+diff --git a/node_modules/stylelint/lib/syntaxes/index.js b/node_modules/stylelint/lib/syntaxes/index.js
+index 7afa0c3..73eaa00 100644
+--- a/node_modules/stylelint/lib/syntaxes/index.js
++++ b/node_modules/stylelint/lib/syntaxes/index.js
+@@ -1,16 +1,13 @@
+ 'use strict';
+ 
+-// Use this require pattern so that syntaxes can be bundled separately
+-const importLazy = require('import-lazy')(require);
```

Comment:
Interesting :thinking: `stylelint` seems to work OK without this patch. Why was this removed - did this have some impact I'm missing, or was this precautionary?

Reply:
`importLazy` breaks our static analysis. removing this patch will work until we regenerate the policy file

Reply:
this should remain as it is